code/__HELPERS/constants.dm

```diff
@@ -1,2 +1,2 @@
-var/global/const/TICKS_IN_DAY = 864000
-var/global/const/TICKS_IN_SECOND = 10
+var/const/TICKS_IN_DAY = 864000
```

Comment:
Because I can't be bothered and `global` like that is incorrect syntax anyways. It wouldn't compile in any sane language.